odoo/http.py

```diff
@@ -149,13 +153,18 @@
 except ImportError:
     from werkzeug.contrib.fixers import ProxyFix
 
+try:
+    from werkzeug.utils import send_file as _send_file
+except ImportError:
+    from .tools._vendor.send_file import send_file as _send_file
```

Comment:
Unless we expect the "official" copy to eventually have useful features, I'd rather we don't do that and just use the vendored copy everywhere, for coherence and clarity.